### DISCUSSION TITLE
Follow Laravel naming conventions for API Resources (specially for Resource Collections)

### DISCUSSION THREAD

**Discussion by jonagoldman** [tailflow/laravel-orion#289] [Help]
The naming convention in Orion is the opposite from Laravel.

**In Laravel, resource collections class names must end with `Collection`.**

From the Laravel documentation on [Customizing The Underlying Resource Class](https://laravel.com/docs/9.x/eloquent-resources#customizing-the-underlying-resource-class) :

> The `$this->collection` property of a resource collection is automatically populated with the result of mapping each item of the collection to its singular resource class.
>
> The singular resource class is assumed to be the collection's class name without the trailing `Collection` portion of the class name.
>
> In addition, depending on your preference, the singular resource class may or may not be suffixed with `Resource`.

This convention is also used by `make:resource` when creating resources. See  [ResourceMakeCommand](https://github.com/laravel/framework/blob/bdbd1aa6145f6a88244eb8f78e08e332dcf0e3b1/src/Illuminate/Foundation/Console/ResourceMakeCommand.php).

**Comment by alexzarbn** [comment]
Hello @jonagoldman,

Thank you for raising the issue. I wasn't aware of the effects on the `$this->collection` property resolution at a time - will see what can be done 🤔
